### ISSUE TITLE
 live boardcast Failed 

### ISSUE THREAD

**Issue by ksandeepvvn** [nickypatson/LiveYoutubeBoardCasting#2]
I Am Getting This Error I created my own Google account and started to broadcast.

When I Click on Create Live Events -> Its Showing This error Live BroadCast Failed

**Comment by rishi420** [comment]
I think you need to change "GoogleService-Info.plist" also @ksandeepvvn

**Comment by nickypatson** [comment]
yeah that right you have to create project then import googleservice-info.plist. Hope this will works!
i am closing this issue trackers

**Comment by ksandeepvvn** [comment]
nickypatson: I Created A New  "GoogleService-Info.plist " and imported in to my project as well, but I concern is it not broadcasting from mobile

**Comment by nickypatson** [comment]
ok give that resposne from youtube i will check.
